snc-library.sh

```diff
@@ -37,9 +37,9 @@ function create_new_release_with_patched_images() {
     ${OC} adm release new -a ${OPENSHIFT_PULL_SECRET_PATH} --from-release=${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE} \
 	    cluster-kube-apiserver-operator=${upstream_registry}/openshift-crc-cluster-kube-apiserver-operator@${kao_image_digest} \
 	    cluster-kube-controller-manager-operator=${upstream_registry}/openshift-crc-cluster-kube-controller-manager-operator@${kcmo_image_digest} \
-	    --to-image=quay.io/crcont/ocp-release:${openshift_version}
+	    --to-image=quay.io/crcont/ocp-release:${openshift_version}-${yq_ARCH}
     # Replace the release image override with crcont image
-    OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=quay.io/crcont/ocp-release:${openshift_version}
+    OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=quay.io/crcont/ocp-release:${openshift_version}-${yq_ARCH}
```

Comment:
An alternative would be to generate the release image once and for all when `cluster-kube-apiserver-operator` and `cluster-kube-controller-manager-operator` are built instead of regenerating it for each snc run.

Reply:
@cfergeau you mean do it beforehand and then run the `snc` script?

Reply:
Yes in the build-patched-kao-kcmo-images.sh
We discussed this before, I forgot if there was anything preventing that or not.